src/rtc/runconfig.py

```diff
@@ -139,19 +133,30 @@ def load_validate_yaml(yaml_path: str, workflow_name: str) -> dict:
     try:
         yamale.validate(schema, data)
     except yamale.YamaleError as yamale_err:
-        err_str = f'Validation fail for {workflow_name} runconfig yaml {yaml_path}.'
+        err_str = (f'Validation failed for runconfig yaml {yaml_path}.')
         logger.error(err_str)
         raise yamale.YamaleError(err_str) from yamale_err
 
     # load default runconfig
     parser = YAML(typ='safe')
-    default_cfg_path = f'{helpers.WORKFLOW_SCRIPTS_DIR}/defaults/{schema_name}.yaml'
-    with open(default_cfg_path, 'r') as f_default:
-        default_cfg = parser.load(f_default)
 
     with open(yaml_path, 'r') as f_yaml:
         user_cfg = parser.load(f_yaml)
 
+    product_type = user_cfg['runconfig']['groups']['primary_executable'][
+        'product_type']
+    if (product_type == STATIC_LAYERS_PRODUCT_TYPE):
+        default_cfg_path = (f'{helpers.WORKFLOW_SCRIPTS_DIR}/defaults/'
+                            f'rtc_s1_sl.yaml')
```

Comment:
What is your decision about the workflow name for the static layer processing after the ADT/SDS meeting last Friday?

Reply:
Thank you, @seongsujeong . It will probably become `RTC_S1_STATIC` rather than `RTC_S1_SL`, but if possible, I'd like to handle this in another PR after I talk to Heresh. In the next PR, I also intend to change the way we define the radar grid for static layers (so that the processing will not be 12x slower).